INSTALL.md

```diff
@@ -2,95 +2,38 @@
 
 In order to run the site in developement mode you need some packages to be installed.
 
-#### Installing `sphinx` into a `virtualenv`
-
-At first, you need to have `sphinx` properly working on your environment. It is needed
-because the next step is to build the documentations locally. 
-
-We suggest to install it into a `virtualenv` and for doing this we first need an environment.
-Let's check if you have correctly installed `virtualenv` by running:
-
-```bash
-which virtualenv
-```
-
-If yes, you will read its path location on the terminal. If not, you can install it following
-the [official instructions](https://virtualenv.pypa.io/en/latest/installation.html).
-
-Now we create an environment that we will use for building the docs using the following command:
-
-```bash
-virtualenv my-sphinx-env
-```
-
-Let's activate the new environment:
-
-```bash
-source my-sphinx-env/bin/activate
-```
-
-Now you will read the name of your environment `(my-sphinx-env)` on the right side of the command line.
-Here we are, now we can proceed with the installation of `sphinx`.
-
-You can do this running following the 
-[official `sphinx` instructions](https://smobsc.readthedocs.io/en/stable/usage/installation.html).
-On Linux or MacOS run:
-
-```bash
-pip install -U sphinx
-```
-
-or on the Windows prompt:
-
-```bash
-pip install -U --pre sphinx
-```
-
-Now the environment is ready to build the documentations.
-
-#### Building the docs
-
-The Qibo site needs the documentations to be generated locally. 
-
-###### Qibo docs
-
-For building `qibo`'s documentation run:
+#### We use `yarn` as dependence
```

Comment:
I will soon switch to `pnpm`, it is much simpler and performant.

When you use Yarn you should also tell which version, since Yarn v1 and Yarn v2+ (also called Yarn Berry) are two completely different package managers.

Reply:
Thank you. I have specified we use `Yarn v1`.